### ISSUE TITLE
Widget aspect

### ISSUE THREAD

**Issue by XanderLeaDaren** [forrestguice/NaturalHour#4]
Hello, here are a few remarks and ideas on how to improve the readability of the widget _when set to 3×2 size_:

· the date is too small, and strangely centred on the VII-th hour, even if it’s not on top
· I think the “rectangles” for hours where I, II, … XII are labelled may become more squarish in order to increase the font size and readability of these number labels
· As a follow-up, 5-min ticks would clearly not be readable… as they already are anyway.

**Comment by forrestguice** [comment]
Agreed that 3x2 size borders on too small. I think making specific adjustments to the smaller widgets shouldn't be a problem (e.g. overriding `clockface_arc_width` in [clockface.xml](https://github.com/forrestguice/NaturalHour/blob/master/app/src/main/res/values/clockface.xml) should allow for more squarish cells).

The date is aligned with the center of daylight to avoid a situation where the outer "night watch ring" might cover it. It happens when the configured timezone is (far) out of alignment with local solar time. I agree it looks weird - maybe there's a better place to put it. I was also thinking the location (lat, lon, etc) should be incorporated into the clockface somehow too. 

..but for the 3x2 widget (and maybe allow smaller?) maybe the solution is to omit detail.
